scripts/borg.exe.spec

```diff
@@ -38,3 +40,16 @@ exe = EXE(pyz,
           strip=False,
           upx=True,
           console=True )
+
+if False:
```

Comment:
rather introduce some flag variable set at the top of the file than hardcoding False here.